### PR TITLE
Features/nes controller

### PR DESCRIPTION
**Pull request by mgoimam**
In Config.pde, set "gamepadConfiguration" to an empty string if you do not have a connected NES controller, otherwise the GameControlPlus configuration UI freezes the application.

### CODE REVIEW

File: GamepadGameInput.pde [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class GamepadGameInput extends GameInput {
+
+  ControlIO control;
+  ControlDevice gpad;
+
+  GamepadGameInput(String config, PApplet applet) {
```
Comment (scjody) [inline GamepadGameInput.pde:6]: Doesn't look like you use `config` yet (and it doesn't work on my mac currently).

**Comment by scjody** [comment]
Other than the config issue this looks good... let me know when you've had a chance to look into that.

**Comment by mgoimam** [comment]
Good catch. Fixed.

**Comment by scjody** [comment]
Looks good.  I'll test it tonight!

**Comment by scjody** [comment]
Works now.  :dancer:  Thanks for helping!

**Comment by scjody** [comment]
err :shipit:

**Pull request merged**